R/install-remote.R

```diff
@@ -31,12 +28,28 @@ install_remotes <- function(remotes, ...) {
 
 # Add metadata
 add_metadata <- function(pkg_path, meta) {
-  path <- file.path(pkg_path, "DESCRIPTION")
-  desc <- read_dcf(path)
+  source_desc <- file.path(pkg_path, "DESCRIPTION")
+  binary_desc <- file.path(pkg_path, "Meta", "package.rds")
+  if (file.exists(source_desc)) {
+    desc <- read_dcf(source_desc)
+
+    desc <- modifyList(desc, meta)
+
+    write_dcf(source_desc, desc)
+  }
```

Comment:
Oh, @jimhester pointed out that both might exist, so this is the right thing to do.  A comment explaining that would be helpful.